Snippets/(thisdir) path to current script.sublime-snippet

```diff
@@ -0,0 +1,6 @@
+<!-- See http://www.sublimetext.com/docs/snippets for more information -->
+<snippet>
+    <content><![CDATA[\$Script:thisDir = Split-Path \$MyInvocation.MyCommand.Path -Parent]]></content>
```

Comment:
Here was my comment on the commit:

This was needed for PS 2.0 in order to get the path to the current script, but since 3.0, you can use ```$PSScriptRoot``` or ```$PSCommandPath``` with scripts and modules. In 2.0, ```$PSScriptRoot``` worked for modules but not scripts. Do we want to keep this snippet in here for getting the script path in 2.0, or should we change it?